composer.json

```diff
@@ -14,9 +14,9 @@
        },
     "require": {
         "php": "^8.0",
-        "lanfix/goaop-framework": "^4.0",
+        "lanfix/goaop-framework": "4.0.4",
```

Comment:
Don't we want to use our package here?

Reply:
I'm going to re-route it in core using vcs repositories:
<img width="667" alt="image" src="https://github.com/sourcetoad/aspect-mock/assets/45333440/05ddd009-993b-47cf-9a3a-2b99445fa37f">

Reply:
I think we should reroute the goap-framework here. Since this is where the package comes in.